Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -5,28 +17,80 @@ $script:discussionClient = $null
 $script:codeReviewClient = $null
 $script:project = $null
 $script:pullRequest = $null
+$script:artifactUri = $null 
 
 #endregion
 
 #region Public
 
+#
+# Initializes the module for posting to the current PR. Usses both vssConnection and env variables to initialize internal actors
+# This function should be called again for a different PR.
+#
 function InitPostCommentsModule
 {
     param ([Microsoft.VisualStudio.Services.Client.VssConnection][ValidateNotNull()]$vssConnection)
     
-    Write-Verbose "InitPostCommentsModule"
+    Write-Verbose "Initializing the PostComments-Module"
```

Comment:
Write-Verbose messages are debug messages. Users need to set to true a build variable named system.debug to get them.